recipe/meta.yaml

```diff
@@ -40,11 +40,13 @@ test:
 about:
   home: https://github.com/googleapis/google-resumable-media-python
   summary: Utilities for Google Media Downloads and Resumable Uploads
-  doc_url: https://googleapis.github.io/google-resumable-media-python/latest/
+  description: Utilities for Google Media Downloads and Resumable Uploads
+  doc_url: https://cloud.google.com/python/docs/reference/google-resumable-media/latest
   dev_url: https://github.com/googleapis/google-resumable-media-python
   license: Apache-2.0
   license_family: APACHE
   license_file: LICENSE
+  license_url: https://github.com/googleapis/google-resumable-media-python/blob/v2.4.0/LICENSE
```

Comment:
no need for a license_url as we have already in package ficense_file.  This tag is superflous